Sources/SimplenoteSearch/Query/SearchQuery.swift

```diff
@@ -60,14 +60,25 @@ public final class SearchQuery: NSObject {
         let keywords = searchText.trimmingCharacters(in: .whitespacesAndNewlines).components(separatedBy: .whitespaces)
 
         for keyword in keywords where keyword.isEmpty == false {
-            guard let tag = keyword.lowercased().suffix(afterPrefix: String.searchOperatorForTags.lowercased()) else {
+            guard let tag = checkForTagOrLocalizedTag(with: keyword) else {
                 items.append(.keyword(keyword))
                 continue
             }
-
+            
             items.append(.tag(tag))
         }
     }
+    
+    private func checkForTagOrLocalizedTag(with keyword: String) -> String? {
+        if let tag = keyword.lowercased().suffix(afterPrefix: String.searchOperatorForTags.tagsKeyword.lowercased()) {
+            return tag
+        }
+        if let tag = keyword.lowercased().suffix(afterPrefix: String.searchOperatorForTags.localizedTagKeyword.lowercased()) {
+            return tag
+        }
```

Comment:
Both ifs in L73 and L76 do the exact same thing. One alternative to drop the duplicated code is to place both Tag Operators in an array, and loop thru them.